### DISCUSSION TITLE
Vote for Infragnistics Controls for Avalonia!

### DISCUSSION THREAD

**Discussion by EtherGhost** [AvaloniaUI/Avalonia#6978] [General]
Vote for Infragnistics Controls for Avalonia! https://m.youtube.com/watch?v=KRvqrQu2964

**Comment by EtherGhost** [comment]
Go to that clip and give my comment a thumb up and write a comment if your own! Show Brian that we are a lots of developers who wants their controls also for Avalonia

**Comment by wieslawsoltes** [comment]
@kvalitetskontrollMarkaryd The link is incorrect, the video is for different framework "Uno Conf Commercial - Infragistics Ultimate UI for Uno Platform"

**Reply by EtherGhost** [reply]
I know, but this is where I asked them to also port their controls to Avalonia

**Reply by wieslawsoltes** [reply]
> I know, but this is where I asked them to also port their controls to Avalonia

Seems like they answered already in comments. I think the best is to contact them directly and let them know that there are many user wanting their controls running on Avalonia framework. Thanks for your support!

**Reply by MikeCodesDotNET** [reply]
I don't think it's a good look to descend on the Uno conf and advocate for Avalonia. 

Perhaps contacting Infragistics directly, either through email, or if you're an existing customer, perhaps you could call your account manager? This would surely be a better demonstration of your eagerness for Infragnistics controls.

**Reply by EtherGhost** [reply]
@MikeCodesDotNET I see your point, but on the other hand, I see nothing wrong with asking for a port for Avalonia as well.  Even on that forum.

**Reply by EtherGhost** [reply]
@MikeCodesDotNET By the way, this is not the Uno conf. This is only the part where Brian is talking about their controls. A clip cut out from Uno conf.

**Reply by mysteryx93** [reply]
Just putting likes on a video comment won't reach the right people. Need more direct contact for the message to get to the right place.

**Comment by mysteryx93** [comment]
As a note, UNO is UWP. No extra work for Infragistics there except testing and perhaps tweaking a few things.

I'm currently porting just one control to Avalonia, and I have to say it's not so easy. Supporting Avalonia would require a lot of work from the part of Infragistics, and although it's been growing a lot recently, it's still a niche market.

But considering that it's getting very mature and that **Microsoft has never been able to produce a true cross-platform UI in 20 years, and still won't with WinUI3 and MAUI**, Avalonia will keep growing in popularity.
